### PR TITLE
Fix bug 1473372: Make Docker setup default in developer docs

### PR DESCRIPTION
**Pull request by mathjazz**
It's time.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,9 +25,9 @@ venv
 .cache/
 coverage/
 
-# For docker
+# Docker
 .docker-build
-docker/config/webapp.env
+docker/config/webapp.env*
```
Comment (adngdb) [inline .gitignore]: I believe that one is incorrect. We have a `webapp.env.template` that we want to track in git. Why did you change this rule?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -24,6 +24,20 @@ If you're not familiar with `Docker <https://docs.docker.com/>`_ and
...
+
+By default, you will have default data loaded for only the Pontoon Intro project.
+If you have a database dump, you can load it into your PostgreSQL database by running::
+
+  $ make loaddb DB_DUMP_FILE=path/to/my/dump
+
+Note that your database container needs to be running while you do that. You
+can start just the postgresql container by runnin::
+
+  $ docker-compose run postgresql
```
Comment (adngdb) [inline CONTRIBUTING.rst]: Might be worth adding `-d` to that command so that the database container runs in the background? I know about it, but others might be frustrated by docker containers showing logs by default.

File: .gitignore (hunk 1 above)
Comment (mathjazz) [reply .gitignore]: `docker/config/webapp.env-e` popped up for me.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -35,7 +35,7 @@ If you have a database dump, you can load it into your PostgreSQL database by ru
 Note that your database container needs to be running while you do that. You
 can start just the postgresql container by runnin::
 
-  $ docker-compose run postgresql
+  $ docker-compose -d run postgresql
```
Comment (adngdb) [inline CONTRIBUTING.rst]: That's not going to work, the `-d` must be at the end.

File: .gitignore (hunk 1 above)
Comment (adngdb) [reply .gitignore]: Huh, OK, that's surprising. Definitely don't see that locally. Can you add specifically `-e` then?

File: CONTRIBUTING.rst (hunk 3 above)
Comment (mathjazz) [reply CONTRIBUTING.rst]: Whoops!

File: .gitignore (hunk 1 above)
Comment (mathjazz) [reply .gitignore]: @adngdb Should I instead list this specific file?

Comment (adngdb) [reply .gitignore]: Yeah that's what I meant.

**Comment by mathjazz** [comment]
@adngdb One final look, please? :)

**Comment by adngdb** [comment]
Le r+!
